src/views/Moderation/Moderation.js

```diff
@@ -178,13 +176,22 @@ export class Moderation extends React.Component {
         try {
             const response = await fetchEvents(queryParams)
 
-            this.setState(state => ({[`${table}Data`]: {
-                ...state[`${table}Data`],
-                events: response.data.data,
-                count: response.data.meta.count,
-                sortBy: columnName,
-                sortDirection: sortDirection,
-            }}))
+            this.setState(state => {
+                const tableData = {...state[`${table}Data`]}
+
+                // reset invalid rows for draft table
+                if (table === 'draft') {
+                    tableData.invalidRows = []
+                }
+
+                return {[`${table}Data`]: {
```

Comment:
This is fine but quite dangerous when refactoring code in the future. It's easy to overlook concatenated object keys so it would be safer to create a wrapper object and then place the individual keys under it:

```js
{
  dataByTable: {
    [table]: ...,
  },
}
```

Reply:
👍